flask/app/rps.py

```diff
@@ -16,3 +16,8 @@ def play():
 @rps.route("/stats")
 def stats():
     return render_template("stats.html")
+
+
+@rps.route("/mainpage")
```

Comment:
what is this page for? I thought we should just send the user to the "play" page

Reply:
just a new index page if they accidentally clicked out of game and no where to enter again

Reply:
what do you mean by "clicked out of game"? how does that happen?

Reply:
right now, if the player were to click on my stats, they wont be able to get back to the index page much less get back into the game.

Reply:
I fixed this by adding a link to play in the navbar in @HamdiaA's PR (which is now merged)

On the index page, there are only login/register/play buttons so unless you want to write code to check if a player is logged in and hide those things I suggest simply redirecting the player to the play page if they try to access the main page.